dataprofiler/data_readers/data_utils.py

```diff
@@ -180,6 +180,59 @@ def read_json_df(data_generator, selected_columns=None, read_in_string=False):
     return json_to_dataframe(lines, selected_columns, read_in_string)
 
 
+def read_json(data_generator, selected_columns=None, read_in_string=False):
+    """
+    This function returns an iterator that returns a chunk of data
```

Comment:
Doesn't this function return a list, not an iterator? There are no yield statements anywhere. You claim its an iterator below in the :returns: section too, but is this still accurate?

Reply:
Fixed the comments, it is not an iterator. Lot of vestigial descriptions from where I copied this.